src/lua/compat.lua

```diff
@@ -51,6 +51,13 @@ initialization or new session creation.
 https://tarantool.io/compat/sql_seq_scan_default
 ]]
 
+local BOX_TUPLE_NEW_ARG_LIST_BRIEF = [[
+Whether `box.tuple.new` should interpret the argument list as an array of tuple
+field — this does not allow passing a tuple format as a second argument.
```

Comment:
Nit: Please, don't use non-ASCII symbols in the code.

Reply:
Fixed.